float64[5] baselink_plane_x
float64[5] baselink_plane_y
float64[5] baselink_plane_z
float64[5] twodim_plane_x
float64[5] twodim_plane_y
float64[5] twodim_plane_z
float64[5] plane_a
float64[5] plane_b
float64[5] plane_c
float64[5] plane_d
